scheduler/crates/infra/src/repos/mod.rs

```diff
@@ -73,8 +73,16 @@ impl Repos {
             .connect(connection_string)
             .await
             .expect("TO CONNECT TO POSTGRES");
-
         info!("DB CHECKING CONNECTION ... [done]");
+
+        if std::env::var("NETTU_SKIP_MIGRATION").is_err() {
```

Comment:
If the env var doesn't exist, we do it

Reply:
@mm-derek Good point

Ideally, here, it's both for local dev and for deployment for us.

The main reason it's not in the CD is that, as this project is open source, it's not integrated with our CI/CD pipelines. More precisely, for releasing, what happens is
- We create a new "Github release" in this repo, which push a new image in the Github packages
  - This happens in this repo only
- Then we manually update the tag we use in our infra when we want to release the new version